csrc/device_lower/pass/allocation.cpp

```diff
@@ -1338,10 +1340,12 @@ class AllocationInserter : public kir::ExprMutator {
               DataType::UInt32,
               lower_utils::getNumThreadsInTensorView(
                   expr->output(0)->as<TensorView>()))));
-      auto sync_init = IrBuilder::create<kir::BlockSync>();
+      auto sync_init = IrBuilder::create<kir::BlockSync>(
```

Comment:
@zasdfgbnm I updated these block syncs to always specify the number of threads in `ComputeWarp`. It avoids problems with epilogue fusions.